pkg/verifier/gcp/gcp_verifier.go

```diff
@@ -16,10 +16,11 @@ import (
 	computev1 "google.golang.org/api/compute/v1"
 )
 
-var (
+const (
 	// TODO find a location for future docker images
-	networkValidatorImage string = "quay.io/app-sre/osd-network-verifier:v0.1.159-9a6e0eb"
```

Comment:
should be referenced by sha256

Reply:
I'm not going to update the GCP image here - it doesn't work for other reasons and don't want to muddy the waters

Reply:
then, just don't touch the GCP file :)

Reply:
Adding a variable can be discussed - I am not a huge fan (mainly because the approach of constantly dupplicating element rather than limiting the amount of variable led to one of the current issues of having partial updates because of the duplication) but there are definitively pros and cons. 
However, the reasoning "it doesn't work for other reasons and don't want to muddy the waters" applies the same here :D . Anyway, this is detail, I will make a MR to aline GCP docker image.. 
For the additional variable topic, may be good to bring it to the team channel. I am not a huge fan of it, but best is to get opinion of others I would say